VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/property/PerformerProperty.kt

```diff
@@ -15,4 +15,8 @@ class PerformerProperty : PolyProperty<BasePerformer>("performer", "Set the perf
     override fun getValues(): List<String> {
         return Performer.getPerformerHandles().toList()
     }
+
+    override fun getAsString(): String {
+        return value.name
+    }
```

Comment:
I think it might be nicer to make a toString() method in BasePerformer.

Reply:
I have chosen a seperate method because implementing toString implies that it represents the whole object, while this method only returns the value